backend/substrapp/tasks/tasks.py

```diff
@@ -338,6 +338,10 @@ def prepare_opener(directory, tuple_):
 
     datamanager = DataManager.objects.get(pk=data_opener_hash)
 
+    # verify that local db opener file exists
+    if not os.path.exists(datamanager.data_opener.path) or not os.path.isfile(datamanager.data_opener.path):
+        raise Exception(f'DataOpener file ({datamanager.data_opener.path}) is missing in local db ')
```

Comment:
I find the `in local db` part to be confusing: why "local db" and not "local filesystem"? 

I'm not sure what "local db" refers to. I would assume that the local db is postgres.

Reply:
I will change it to local storage